pages/whats_new.md

```diff
@@ -0,0 +1,31 @@
+---
+title: What's New
+layout: content
+permalink: /about/whats-new/
+breadcrumb: About
+---
+
+The initial USEITI beta site launched in early 2015, offering visualizations that showed how natural resources generate revenue across the country and where the money goes.
+
+This site, which launched in December 2015, makes the data easier to use, improves the site structure, and provides context about the government's role in extractive industries.
+
+We've also added new information and datasets to explore:
+
+* **Production data:** Sales volumes and revenue have long been used to estimate how much of each resource was produced. Here, for the first time, you can find more precise data on what resources were produced on federal lands. [Explore federal production data.][~1]
```

Comment:
It isn't a convention that I'm aware of. I did it this way because having as short a string as possible seemed consistent with the goal of not interfering with the content text. Since we are already using the `[^1]` notation for footnotes, it seemed like using the same method with a different symbol made sense. @coreycaitlin from a content management perspective, does one method seem preferable?